pkg/controller/bookkeepercluster/bookie.go

```diff
@@ -124,20 +125,38 @@ func makeBookiePodSpec(bk *v1alpha1.BookkeeperCluster) *corev1.PodSpec {
 		// default value if user did not set ledgerDirectories in options
 		ledgerDirs = append(ledgerDirs, "/bk/ledgers")
 	}
+	if _, ok = bk.Spec.Options["ledgerSubPath"]; ok {
```

Comment:
Do we need to change the CRD to be able to take extra options on the bk cluster resours? 
I'm not sure how permissive the acceptance hook validation is.

Reply:
Changes is CRD are needed only if a new field is added to the spec or status. Here since the new parameter being added is part of `options` field within is already part of the bookkeeper cluster spec, no changes to the CRD would be required